addons/website/models/ir_ui_view.py

```diff
@@ -151,7 +153,7 @@ def _sort_suitability_key(self):
                 * then if the view has no set website
         """
         self.ensure_one()
-        context_website_id = self.env.context.get('website_id', 1)
+        context_website_id = self.env.context.get('website_id', None)
```

Comment:
can we remove this from this commit ? since improved in futur commit

Reply:
except if it will fail your test.

Reply:
Yes it fails the 
```python
self.assertEqual(inherit_view.active, True, "_view_obj should return the generic one")
```